### ISSUE TITLE
Length of the sections

### ISSUE THREAD

**Issue by mgifford** [canada-ca/digital-playbook-guide-numerique#213]
I think there's lots of good stuff here, but it needs to be trimmed down a bit.  Some of those resources should be put elsewhere. Lots of good 101 things, but that should probably live somewhere else. 

There are a few that are of a similar length. Ultimately the playbook should be punchy & readable. Supporting links and material should be housed and possibly maintained elsewhere. 

Also, I hope there are images, colors or visuals going ahead.

**Comment by pjackson28** [comment]
I agree that it should be punchy and readable, but we also should be careful to not water it down too much and to ensure the guidance is relevant to real-world scenarios. We are trying to get the best of both worlds through a "views" approach, where the main playbook is a comprehensive knowledge base and each "view" is a subset of that knowledge targeted to a specific task/scenario (that would also be  punchy and readable). For instance, there could be a "view" for creating a service, setting up a help desk or building a website, where the "view" would include the steps for each of those tasks, with relevant guidance mapped to those steps from the playbook knowledge base.

All the material in the playbook is now stored in a JSON file and we are currently experimenting with some "views" and how to slice and dice the information. Suggestions welcome  for tasks/scenarios that would be good candidates for "views" and how those "views" should be presented. Those could be good starting points for testing with users since we want to make sure that what we provide is as relevant and useful as possible. 

As for images, colors and visuals, they could certainly be included as part of the content. Suggestions welcome for where we can improve in those areas. It would be good to come up with a good draft that can be tested with users, where the use of images, colors and visuals can be tweaked to best meet their needs.

**Comment by mgifford** [comment]
Thanks Paul.  I'm not sure how to achieve this goal. 

As far as images, I do think it can be useful to pull ideas from https://thenounproject.com/

They have lots of options under a CC license that you can use.  Just need to provide attribution if you don't want to pay for them.  Nice example of dual licensing model and they come in SVG & PNG files.

**Comment by pjackson28** [comment]
I'll be working with others on creating some task-based "views" to help demonstrate the concept. I admit it is quite difficult to describe how views work through text alone. Perhaps it will be easier for you or others to come up with suggestions once we have a few working examples.

As for the link you provided, that seems to be a good resource for icons. Do they provide guidance on when it is best to use the icons? I took a quick look and didn't notice any such guidance. Maybe I'll look around at some design systems to see what they suggest (to help avoid going too far with icons).

**Comment by mgifford** [comment]
I don't think I've seen any guidance about this.

**Comment by thomasgohard** [comment]
The general guidance is that when you use an icon it must be accompanied by a text label. That's because you can't assume that people will understand what the icon means. People rarely know what an icon represents outside a few common case. And, even then, the common cases are often region- or domain-specific and not universal.
